src/saddle.ts

```diff
@@ -84,6 +86,11 @@ export async function getSaddle(network, trace=false, quiet=false): Promise<Sadd
     return await deployContract(web3 || network_config.web3, network_config.network, contractName, args, network_config, network_config.defaultOptions, options);
   }
 
+  // TODO: Should this call into CLI code or maybe we should factor into a general util module?
```

Comment:
Would be convenient to have all saddle functionality first-class accessible as a library

Reply:
Yeah, there's a lot here that could be cleaned up. It's hard because every client wants to use it in a different way. It's one of the hard things about building a consistent Ethereum wrapper for anything.